PWGLF/Tasks/Nuspex/nuclei_in_jets.cxx

```diff
@@ -1018,21 +1056,22 @@ struct nuclei_in_jets {
     }
   }
 
-  void processAntipJet(o2::aod::McCollisions const& mcCollisions,
-                       aod::McParticles const& mcParticles)
+  void processAntipJet(o2::aod::McCollisions const& mcCollisions, aod::McParticles const& mcParticles)
   {
 
+    // Loop over MC Collisions
     for (const auto& mccollision : mcCollisions) {
 
+      // Selection on z_{vertex}
       if (abs(mccollision.posZ()) > 10)
         continue;
 
-      auto mcParticles_per_coll =
-        mcParticles.sliceBy(perMCCollision, mccollision.globalIndex());
+      // MC Particles per Collision
+      auto mcParticles_per_coll = mcParticles.sliceBy(perMCCollision, mccollision.globalIndex());
 
       // Reduced Event
       std::vector<int> particle_ID;
-      int leading_ID = 0;
+      int leading_ID;
```

Comment:
please initialize leading_ID=0 as the same bug broke today's tag already :)

Reply:
fixed!